modules/vba_documents/spec/factories/upload_submissions.rb

```diff
@@ -51,6 +51,12 @@
       detail { 'Upload rejected' }
     end
 
+    trait :status_emms_error do
+      status { 'error' }
+      code { 'DOC202' }
+      detail { 'image failed to process' }
+    end
```

Comment:
Similar comment to above, recommend `:status_upstream_error`.

Reply:
I like the realistic error detail you used below, `Upstream status: Errors: ERR-EMMS-FAILED, Corrupted File detected.`. Did you consider using that here?

Reply:
done